generators/boilerplate/index.js

```diff
@@ -6,6 +6,14 @@ module.exports = generators.Base.extend({
   constructor: function () {
     generators.Base.apply(this, arguments);
 
+    // destination path
+    this.option('path', {
```

Comment:
Oh… I just remembered `projectRoot` option is already used on `node:app` and if I am not wrong for a different end.